### ISSUE TITLE
inconsistency between MANGO paper and its code

### ISSUE THREAD

**Issue by LianaWang** [hikopensource/DAVAR-Lab-OCR#101]
The instance mask attention(IMA) module which predicts H*W*S^2 instance masks seems not used in the MANGO implementation according to the config file and mango.py, and an additional SOLO category branch is used instead. 
So is the config file wrong or did you just implement it this way?

**Comment by qiaoliang6** [comment]
Yes, the current released code was reimplemented from the original version. (See explanation in #61). 

BTW, are you studying with Prof. Xi Li? We can have more privately communication through Wechat.
